Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -1,163 +1,137 @@
     for traffic in response:
-        hr_output.append({
-            "Source IP": traffic.get('src', {}).get('ip'),
-            "Destination IP": traffic.get('dst', {}).get('ip'),
-            "Destination Workload Hostname": traffic.get('dst', {}).get('workload', {}).get('hostname'),
-            "Service Port": traffic.get('service', {}).get('port'),
-            "Service Protocol": protocol,
-            "Policy Decision": traffic.get('policy_decision'),
-            "State": traffic.get("state"),
-            "Flow Direction": traffic.get('flow_direction'),
-            "First Detected": arg_to_datetime(traffic.get('timestamp_range', {}).get('first_detected', '')).strftime(  # type: ignore # noqa
-                HR_DATE_FORMAT),  # type: ignore # noqa
-            "Last Detected": arg_to_datetime(traffic.get('timestamp_range', {}).get('last_detected', '')).strftime(  # type: ignore # noqa
-                HR_DATE_FORMAT)
-        })
-
-    headers = ["Source IP", "Destination IP", "Destination Workload Hostname", "Service Port", "Service Protocol",
-               "Policy Decision", "State", "Flow Direction", "First Detected", "Last Detected"]
+        hr_output.append(
+            {
+                "Source IP": traffic.get("src", {}).get("ip"),
+                "Destination IP": traffic.get("dst", {}).get("ip"),
+                "Destination Workload Hostname": traffic.get("dst", {}).get("workload", {}).get("hostname"),
+                "Service Port": traffic.get("service", {}).get("port"),
+                "Service Protocol": protocol,
+                "Policy Decision": traffic.get("policy_decision"),
+                "State": traffic.get("state"),
+                "Flow Direction": traffic.get("flow_direction"),
+                "First Detected": arg_to_datetime(
+                    traffic["timestamp_range"]["first_detected"]).strftime(HR_DATE_FORMAT),  # type: ignore
+                "Last Detected": arg_to_datetime(
+                    traffic["timestamp_range"]["last_detected"]).strftime(HR_DATE_FORMAT),  # type: ignore
+            }
+        )
 
+    headers = list(hr_output[0].keys())
     return tableToMarkdown("Traffic Analysis:", hr_output, headers=headers, removeNull=True)
 
 
-def prepare_hr_for_illumio_virtual_service_create_command(result: dict) -> str:
-    """Prepare human-readable output for illumio virtual service create command.
+def prepare_virtual_service_output(response: dict) -> str:
+    """Prepare human-readable output for virtual-service-create command.
 
     Args:
-        result: result returned after creating Virtual Service.
+        response: result returned after creating Virtual Service.
 
     Returns:
-        human-readable output.
+        markdown string to be displayed in the war room.
     """
-    table = {
-        num: name[len(SOCKET_PREFIX):]
-        for name, num in vars(socket).items()
-        if name.startswith(SOCKET_PREFIX)
-    }
+    table = {6: "tcp", 17: "udp"}
```

Comment:
Make this a constant, or better yet an enum.

Reply:
Created an enum.